assets/styles/layout/_options.scss

```diff
@@ -28,6 +28,26 @@
     }
   }
 
+  .options__filter .dropdown__menu {
+    /* Positioning for dropdown menus inside .options__main */
+    left: 0;
+    right: auto; /* Reset the right property */
+    top: 100%; /* Position it below the trigger element */
+    transform: none;
+    min-width: 10rem;
+    
+  }
+  
+  .options__filter .dropdown:hover .dropdown__menu,
+  .options__filter .dropdown:focus-within .dropdown__menu {
+    /* Positioning for dropdown menus inside .options__main */
+    left: 0;
+    right: auto; /* Reset the right property */
+    top: 100%; /* Position it below the trigger element */
+    transform: none;
+    min-width: 10rem;
+  }
```

Comment:
These 2 are the same so they can be merged

Reply:
Done